blocks/community_product_list/form.php

```diff
@@ -96,6 +97,21 @@ class="existing-select2 select2-select" autocomplete="off" style="width: 100%" p
             <?php
             } ?>
 
+            <?php
+            $productmanufacturers = array("0" => t("None"));
+            foreach ($manufacturersList as $productmanufacturer) {
+                $productmanufacturers[$productmanufacturer->getID()] = $productmanufacturer->getName();
+            }
+            ?>
+            <?php if (!empty($productmanufacturer)) {
+            ?>
+            <div class="form-group">
+                <?= $form->label('mID', t('Filter by Product Manufacturer')); ?>
```

Comment:
change to 'Filter by Brand / Manufacturer'  (to match what it's called on the edit product form)